ui/components/app/assets/asset-list/network-filter/network-filter.tsx

```diff
@@ -137,27 +131,29 @@ const NetworkFilter = ({ handleClose }: SortControlProps) => {
             </Text>
           </Box>
           <Box display={Display.Flex} alignItems={AlignItems.center}>
-            {chainsToShow
-              .slice(0, 5) // only show a max of 5 icons overlapping
-              .map((chain, index) => {
-                const networkImageUrl =
-                  CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[
-                    chain as keyof typeof CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP
-                  ];
-                return (
-                  <AvatarNetwork
-                    key={chainId}
-                    name="All"
-                    src={networkImageUrl ?? undefined}
-                    size={AvatarNetworkSize.Sm}
-                    // overlap the icons
-                    style={{
-                      marginLeft: index === 0 ? 0 : '-20px',
-                      zIndex: 5 - index,
-                    }}
-                  />
-                );
-              })}
+            <InfoTooltip
+              position="bottom"
+              contentText={FEATURED_NETWORK_NAMES.join(', ')}
```

Comment:
The user may not have all 9 of these networks added.  I think we should show just the subset of the featured networks they have added, since that's what we're actually querying

Reply:
Ahh, makes sense. Let me revisit

Reply:
this would apply to the icons and text